ingest_graph_validator/hydrators/ingest_hydrator.py

```diff
@@ -120,10 +120,9 @@ def get_edges(self):
                             if relationship_name == 'DERIVED_BY_PROCESSES':
                                 edges.append(Relationship(end_node, 'DUMMY_EXPERIMENTAL_DESIGN', start_node))
 
+                            self._logger.debug(f"({start_node['id']})-[:{relationship_name}]->({end_node['id']})")
```

Comment:
Might be good to check first why end_node seems to be uninitialised/missing for a specific dataset